.github/workflows/run-grchombo-tests-gcc.yml

```diff
@@ -24,13 +24,13 @@ jobs:
     - name: Install Chombo dependencies
       run: |
         sudo apt-get update
-        sudo apt-get -y --no-install-recommends install csh make gfortran-8 g++-8 cpp-8 libhdf5-dev libhdf5-openmpi-dev openmpi-bin libblas-dev liblapack-dev libgetopt-complete-perl ssh
+        sudo apt-get -y --no-install-recommends install csh gfortran-10 g++-10 cpp-10 libhdf5-dev libhdf5-openmpi-dev openmpi-bin libblas-dev liblapack-dev libgetopt-complete-perl
```

Comment:
Why was make removed from the list of installed packages? Is it pulled as a dependency of one of packages? What about ssh? The fact that I'm asking may indicate that reason for these changes should be given in commit message.

Reply:
This was part of the update to Ubuntu 20.04. As it turns out these packages are already installed on the runners running this OS version whereas this wasn't the case on 18.04.

Reply:
I have update the commit message accordingly.